pkgs/by-name/do/dotnet-dbus/package.nix

```
@@ -0,0 +1,23 @@
+{ buildDotnetGlobalTool, lib }:
+
+buildDotnetGlobalTool {
+  pname = "dotnet-dbus";
+  version = "0.21.2";
+
+  nugetName = "Tmds.DBus.Tool";
+  nugetHash = "sha256-PUJ0hvKJT/gReWkiBffw8B/oSPG/rNXISidwEQF8uzA=";
+
+  meta = with lib; {
+    description = "dotnet-dbus - Tmds.DBus tool for generating C# code from D-Bus XML interface descriptions";
```

```suggestion
    description = "Tool for generating C# code from D-Bus XML interface descriptions";
```
description must not repeat the package name.